### PR TITLE
CON-2128: Use position sticky when supported

### PR DESCRIPTION
**Pull request by lukaszkonieczny**

### CODE REVIEW

File: extensions/wikia/ArticleNavigation/scripts/sticky.js [hunk 1]
```diff
@@ -88,13 +88,38 @@ require([
...
-	// Sticky navigation
-	stickyElementObject.init({
-		sourceElement: navigationElement,
-		alignToElement: boundBoxElement,
-		topFixed: globalNavigationHeight + additionalTopOffset,
-		minWidth: layout.breakpoints.smallMin,
-		adjustFunc: adjustValueFunction,
-		adjustPositionFunc: adjustPositionFunction
-	});
+	function adjustTopPosition() {
```
Comment (vforge) [inline extensions/wikia/ArticleNavigation/scripts/sticky.js]: As you're using ```navigationElement``` in ```isPositionStickySupported``` is it really needed to have separate function that is called only one time? Is there any more reason that I don't see here?

**Comment by vforge** [comment]
lgtm

**Comment by hakubo** [comment]
@lukaszkonieczny @vforge is this still needed?

**Comment by vforge** [comment]
Nope, I closed this pull.

**Pull request closed without merging**